lib/plug/session/cookie.ex

```diff
@@ -37,7 +37,17 @@ defmodule Plug.Session.COOKIE do
 
     * `:serializer` - cookie serializer module that defines `encode/1` and
       `decode/1` returning an `{:ok, value}` tuple. Defaults to
-      `:external_term_format`.
+      `:external_term_format`;
+
+    * `:active_secret_key_bases` - a list of secret key bases that are active
+      for decrypting and verifying a cookie in addition to `:secret_key_base`.
+      These are not used for encrypting or signing. Use this when rotating the
+      secret key base so that cookies encrypted/signed by a key base other than
+      the current `:secret_key_base` are not invalidated and can be read. The
+      list will be tried in order after the `:secret_key_base` to
+      decrypt/verify. When a cookie is updated the `:secret_key_base` is always
+      used to encrypt/sign. Can be either a list of binaries or an MFA
+      returning a list of binaries. Defaults to `[]`;
```

Comment:
Not sure how secure it is to support non-MFA because might be too easy for someone to accidentally leak. Would anyone be using this feature without an MFA?

Reply:
People may be mounting plugs dynamically and reading from elsewhere. So it  is fine to support a non-MFA but the MFA should be the main way.